ntc_templates/templates/cisco_asa_show_cpu_usage_detailed.textfsm

```diff
@@ -0,0 +1,14 @@
+Value CORE (Core\s\d+)
+Value CPU_5_SEC (\d+\.\d+)
+Value CPU_5_SEC_DP (\d+\.\d+)
+Value CPU_5_SEC_CP (\d+\.\d+)
+Value CPU_1_MIN (\d+\.\d+)
+Value CPU_1_MIN_DP (\d+\.\d+)
+Value CPU_1_MIN_CP (\d+\.\d+)
+Value CPU_5_MIN (\d+\.\d+)
+Value CPU_5_MIN_DP (\d+\.\d+)
+Value CPU_5_MIN_CP (\d+\.\d+)
+
+
+Start
+  ^${CORE}\s+${CPU_5_SEC}\s+\(${CPU_5_SEC_DP}\s+\+\s+${CPU_5_SEC_CP}\)\s+${CPU_1_MIN}\s+\(${CPU_1_MIN_DP}\s+\+\s+${CPU_1_MIN_CP}\)\s+${CPU_5_MIN}\s+\(${CPU_5_MIN_DP}\s+\+\s+${CPU_5_MIN_CP}.+ -> Record
```

Comment:
Please add the error handling as well. Such as https://github.com/networktocode/ntc-templates/blob/master/ntc_templates/templates/alcatel_aos_show_vlan.textfsm#L18

Reply:
Thank you for your review. I added the error handling.
Please review it.